docs/application/native/guides/text-input/autofill-service.md

```diff
@@ -0,0 +1,335 @@
+# Autofill Service
+
+Autofill Service application is a service application to store the user input and provides the data previously entered by the user. Developers can implement the Autofill service in their own application to serve the Autofill information.
+
+The main features of the Autofill service API include:
+
+- Authentication Information
```

Comment:
Sending and receiving authentication information

Reply:
At this moment, I've used "Receive authentication information" expression.
It's because the "sending" thing is the request to get authentication not "authentication information". How is your opinion?

Reply:
@imfine98, 
 sure I agree with your opinion.